test/integration/integration_test.go

```diff
@@ -29,6 +29,7 @@ const (
 	FakeImageScripts                = "sti_test/sti-fake-scripts"
 	FakeImageScriptsNoSaveArtifacts = "sti_test/sti-fake-scripts-no-save-artifacts"
 	FakeImageNoTar                  = "sti_test/sti-fake-no-tar"
+	FakeImageOnBuild                = "sti_test/sti-fake-onbuild"
```

Comment:
I would have expected this to be the "sti_test/sti-fake-onbuild-source" image (prior to my suggested renaming).

the idea of ONBUILD type builds is that you've got a builder image which contains ONBUILD instructions that should be run as part of the STI build (which is accomplished by performing a "docker build" using the builder image as the FROM)